pkg/diagnostics/service_monitoring.go

```diff
@@ -140,6 +153,28 @@ func newServiceMetrics() *serviceMetrics {
 			"The number of requests blocked by the global action specified in the access control policy.",
 			stats.UnitDimensionless),
 
+		// Service Invocation
+		serviceInvocationRequestSentTotal: stats.Int64(
+			"runtime/service_invocation/req_sent_total",
+			"The number of requests sent by service invocation.",
```

Comment:
Nit: not sure if "by service invocation" or another preposition... "via", "through", "in"?

Reply:
updated, ty